### ISSUE TITLE
[finnish] wants notation used for config properties coming from RxJS & 3rd party libs

### ISSUE THREAD

**Issue by ChrisMBarr** [JasonWeinzierl/eslint-plugin-rxjs-x#338]
I have an Angular httpInterceptor that will retry an error after a short delay.  I make use of the RxJS `retry` which has a config property named `delay` which can accept an observable. Docs: https://rxjs.dev/api/index/interface/RetryConfig

### Problem
The `finnish` rule is saying that I need to name this property `delay$` but that is not something I have control over since it comes from a 3rd party library.

### Expected Result
Observable properties (but anything really) from a 3rd party library, especially RxJS itself, should not trigger a lint issue telling me to rename something I do not have the ability to rename.

### Code
I am using this in my config
```
'rxjs-x/finnish': ['warn', { strict: true }],
```

And here is the code in context
```ts
export const httpErrorInterceptor$: HttpInterceptorFn = (req, next$) => {
  environmentService = inject(EnvironmentService);

  const clonedRequest = req.clone({
    withCredentials: req.url.includes(environmentService.webServerUrl),
  });

  return next$(clonedRequest).pipe(
    retry({ delay: genericRetryStrategy() }),
    //      ~~~~~  Error reported here
    catchError((error: HttpErrorResponse) => {
      const code = getCode(error);

      if (code === (HttpStatusCode.Unauthorized)) return handleUnauthorized$(error);
      if (code === (HttpStatusCode.ServiceUnavailable)) return handleOutageMode$();

      return catchAll$(clonedRequest.url, error);
    }),
  );
};
```

### Screenshots

VSCode displaying the error
<img width="1012" height="254" alt="Image" src="https://github.com/user-attachments/assets/90e940bd-64f7-454c-9c6a-d763fe6d830b" />

Obviously, this will not work since `delay$` is not a property that exists

<img width="943" height="301" alt="Image" src="https://github.com/user-attachments/assets/642ed25f-4260-44b1-bd09-c1e2e80f50fb" />

-----

Another example would be on an Angular route config.  
```ts
[
  {
    path: '',
    title: '',
    pathMatch: 'full',
    redirectTo: (): MaybeAsync<string> => {
//  ~~~~~~~~~~ Error reported here
    //Do some logic in here
    },
  },
  {
    path: 'whatever',
    title: someTitleResolver$
//  ~~~~~ Error reported here
  }
]
```

**Comment by ChrisMBarr** [comment]
Hmm yeah I see the issue. I assume there's probably no way to tell if a property name originates from the code in the project being linted, or from a 3rd-part NPM package?

Can there be addition whitelisted/default excluded properties?  Since this is an RxJS focused set of lint rules it would make sense to never flag things coming out or RxJS itself


Thankfully for my use cases, I only ran into this issue in a small number of locations. I'd still prefer to leave the `properties` option enabled for now and just disable things as I run into them.
